app/src/main/java/org/simple/clinic/home/patients/PatientsScreen.kt

```diff
@@ -102,6 +105,19 @@ open class PatientsScreen(context: Context, attrs: AttributeSet) : RelativeLayou
         .observeOn(mainThread())
         .takeUntil(screenDestroys)
         .subscribe { uiChange -> uiChange(this) }
+
+    illustrationImageView.setImageResource(illustrationResourceId())
+  }
+
+  private fun illustrationResourceId(): Int {
+    val today = LocalDate.now()
+    val may_11_2019 = LocalDate.of(2019, Month.MAY, 11)
```

Comment:
One more piece of feedback I have is that the variable name `may_11_2019` is pointless since I have no idea why this particular day is important. I can suggest changing the code that make it more obvious like so:

```kotlin
val today = LocalDate.now(userClock)
val worldHypertensionDay = today.withMonth(Month.MAY.value).withDayOfMonth(17)
val dateToShowBannerFrom = worldHypertensionDay.minusDays(6L)

return when(today) {
  in dateToShowBannerFrom..worldHypertensionDay -> R.drawable.ic_homescreen_world_hypertension_day
  else -> R.drawable.illustrations_homescreen
}
```

Reply:
This also handles the case where we need to show the banner the next year as well.

Reply:
done